src/modules/lodash.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/prefer-default-export */
+import _ from 'lodash';
+
+export function component() {
```

Comment:
- [ ] [OPTIONAL] Thank you for remove all other `eslint-disables` you only have one here now. The way you could fix this is by making `component` a default export. 
## something like this
```suggestion

import _ from 'lodash';

export default function component() {
```

- [ ] [OPTIONAL] also it seems that you are not using this file. Since it came with the default webpack setup you can just remove this file and also remove `lodash` as a dependency on your package.json